docs/source/spec/core/stream-traits.rst

```diff
@@ -394,7 +303,7 @@ Initial messages
 ----------------
 
 An *initial message* is comprised of the top-level input or output members
-of an operation that are not targeted by the ``eventStream`` trait. Initial
+of an operation that do not target the event stream union. Initial
 messages provide an opportunity for a client or server to provide metadata
 about an event stream before transmitting events.
 
```

Comment:
Let's just remove the "important" note below. All protocols should support initial messages, so it's strange to call it out here instead of just in the protocol documentation itself.